insights/tests/client/test_client.py

```diff
@@ -49,6 +49,7 @@ def test_version():
         sys.argv = tmp
 
 
+@pytest.mark.skip(reason="Mocked paths not working in QE jenkins")
```

Comment:
Maybe consider using [NamedTemporaryFile](https://docs.python.org/ja/2.7/library/tempfile.html#tempfile.NamedTemporaryFile). I‘m not sure whether it’d work, since it actually writes files to _/tmp_ just as you do, but it might be worth trying.